### ISSUE TITLE
Accessibility: Refunds Sample Page Content includes skipped heading level

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43602]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The sample content on the Refund policy page skips a heading level. See:
https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce/includes/class-wc-install.php#L2286

"Overview" is set to an H3 when it should be an H2. Having correct heading structure helps screen reader users navigate pages. Sample content should use the correct heading level so that website owners don't have to fix it if they use the sample content to create their own page.

### Expected behavior

Headings should create a meaningful hierarchy for content on the page without skipping heading levels. The first heading on a page after the page title (H1) should always be an H2.

### Actual behavior

A heading 3 is used instead of a heading 2.

### Steps to reproduce

See https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce/includes/class-wc-install.php#L2286

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by dinhtungdu** [comment]
> A heading 3 is used instead of a heading 2.

Make sense to me. Thank you for the report!

**Comment by amesplant** [comment]
@dkotter  It looks like this is no longer an issue and should be closed.
https://github.com/woocommerce/woocommerce/blob/d0e056729190feaab1ea3f3b1eddf2fd8bbdb65a/plugins/woocommerce/includes/class-wc-install.php#L2505

**Comment by dinhtungdu** [comment]
> It looks like this is no longer an issue and should be closed.

Indeed. Thanks for the head up! I'm closing this issue now.
